docs/release-notes/release-notes-0.13.1.md

```diff
@@ -25,6 +25,9 @@
   template and GitHub Actions check to ensure that they are always updated 
   unless the `no-changelog` label is added to the PR.
 
+* [Add tlsextraip & tlsextradomain to
+  conf](https://github.com/lightninglabs/lightning-terminal/pull/778)
+
```

Comment:
missing your name in contributors list below!

Reply:
Ah thanks! TIL :)!